quickstarts/microsoft.machinelearningservices/machine-learning-job-create-or-update-job/main.bicep

```diff
@@ -0,0 +1,52 @@
+@description('Specifies the name of the Azure Machine Learning workspace where the command job will be created.')
+param workspaceName string
+
+@description('Specifies the location for all resources.')
+param location string = resourceGroup().location
+
+@description('Specifies the name of the Azure Machine Learning compute instance/cluster on which job will be run.')
+param computeName string
+
+@description('Specifies the name of the Azure Machine Learning experiment under which job will be created.')
+param experimentName string
+
+@description('The name for the storage account to created and associated with the workspace.')
+param storageAccountName string
+
+@description('Specifies the name of the Azure Machine Learning job to be created.')
+param jobName string
+
+// creating codeVersion resource
+module codeVersion 'modules/codeversion.bicep' = {
+  name: 'blob'
+  params: {
+    location: location
+    workspaceName: workspaceName
+    storageAccountName: storageAccountName
+  }
+}
+
+// param environmentId string
+@description('Specifies the curated environment to run sweep job.')
+param environmentName string = 'AzureML-lightgbm-3.2-ubuntu18.04-py37-cpu'
+
+// fetching existing curated environment resource object
+resource environmemt 'Microsoft.MachineLearningServices/workspaces/environments@2022-05-01' existing = {
```

Comment:
any reason we can make this a reference to the version that's needed below?  then we can remove the resourceId function and use a simple symbolic reference...

Reply:
Actually I think you can remove this altogether and then use the module output below (will add the comment there).